.bumpversion.cfg

```
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.15.0
+current_version = 2.16.0
```

```suggestion
current_version = 2.15.0
```

We bump versions in the github action.